config.ini

```diff
@@ -1,10 +1,12 @@
 [http]
-bind_to_address = 0.0.0.0
-bind_to_port = 4445
+bind_to_address=0.0.0.0
```

Comment:
I see the spaces have been removed here. ConfigParser by default uses spaces between the `=`. It can work without spaces, but it's probably best to keep them.